src/main/java/org/terasology/corerendering/rendering/dag/nodes/ShadowMapNode.java

```diff
@@ -48,9 +52,9 @@
 public class ShadowMapNode extends ConditionDependentNode implements PropertyChangeListener {
     public static final SimpleUri SHADOW_MAP_FBO_URI = new SimpleUri("engine:fbo.sceneShadowMap");
     private static final ResourceUrn SHADOW_MAP_MATERIAL_URN = new ResourceUrn("engine:prog.shadowMap");
-    private static final int SHADOW_FRUSTUM_BOUNDS = 500;
-    private static final float STEP_SIZE = 50f;
+    private static final int SHADOW_FRUSTUM_BOUNDS = 200;
```

Comment:
shrink the frustrum to increase the resolution of individual texels. maybe this could be better managed with some dynamic scheme.